test/common/testlib.py

```diff
@@ -1299,7 +1299,8 @@ def assert_pixels_in_current_layout(
         scroll_into_view: str | None = None,
         wait_animations: bool = True,
         wait_delay: float = 0.5,
-        chrome_hack_double_shots: bool = False
+        chrome_hack_double_shots: bool = False,
+        fuzzyness: float = 20
```

Comment:
As this is API: It's spelled "fuzziness". But perhaps let's make it a bit more descripitive, like `abs_tolerance`?

This is the number of changed pixels it tolerates, so it is really an `int`, not a float. We may at some point change to a percentage (which IMHO makes sense) and default to 0.05% (we currently allow 0.02% for a 300x300 dialog-size-ish screenshot), then it becomes a float.